### PR TITLE
Issue 3

### PR DESCRIPTION
**Pull request by Mohare993**
- [x] Merge is going to correct branch.
- [x] Code is well formated (readable)
- [x] Code uses sensible variable/method names (readable)
- [x] Peer Reviewed
- [ ] Reviewed by Tadas

### CODE REVIEW

**Review by JayKL — COMMENTED** [review]
no problems detected

File: app/models/UserDatabase.scala [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package models
+
+import javax.inject.Inject
+
+@javax.inject.Singleton
+class UserDatabase @Inject()() {
+  def lookupUser(u: User): Boolean = {
```
Comment (tvaidotas) [inline app/models/UserDatabase.scala]: `u` is not a good name

File: app/models/UserDatabase.scala [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    //TODO query your database here
+    if (u.username == "foo" && u.password == "foo") true else false
```
Comment (tvaidotas) [inline app/models/UserDatabase.scala]: don't use literal values like `"foo"` they should come from constants file

**Comment by tvaidotas** [comment]
there's no tests for any of this

File: app/models/UserDatabase.scala (hunk 2 above)
Comment (Mohare993) [reply app/models/UserDatabase.scala]: I've removed this file as it was accidentally pushed to my branch.

**Comment by Mohare993** [comment]
tests have been added

**Review by tvaidotas — APPROVED** [review]

**Pull request merged**